src/Workspaces/Core/Portable/Workspace/ProjectSystem/VisualStudioAnalyzer.cs

```diff
@@ -17,7 +17,7 @@ internal sealed class ProjectAnalyzerReference(string fullPath, IProjectSystemDi
     // NOTE: It is important that we share the same shadow copy assembly loader for all VisualStudioAnalyzer instances.
     // This is required to ensure that shadow copied analyzer dependencies are correctly loaded.
     private static readonly IAnalyzerAssemblyLoader s_analyzerAssemblyLoader =
-        new ShadowCopyAnalyzerAssemblyLoader(Path.Combine(Path.GetTempPath(), "VS", "AnalyzerAssemblyLoader"));
+        DefaultAnalyzerAssemblyLoader.CreateNonLockingLoader(Path.Combine(Path.GetTempPath(), "VS", "AnalyzerAssemblyLoader"));
```

Comment:
I considered using the workspace service here, but that uses a different path in temp to put them, so I didn't.  cc @jasonmalinowski

Reply:
Does this still run in VS Code? Other than the initializer running, does anything use it?

Reply:
resolved by adding a version of the service exported for the host workspace (using the VS/AnalyzerAssemblyLoader path)